test/test_ops.py

```diff
@@ -823,11 +826,7 @@ def test_jit_alias_remapping(self, device, dtype, op):
         # Below we prepare strings of args/kwargs with and without type annotations.
         # These strings are inserted into function template strings which is then torch scripted.
         # - args string is ["t0"] corresponding to the "input" tensor required by the op
-        # - args_annot_kw is the string for the template function signature, for example,
-        # ["t0", "s0: float", "s1: bool", "max: float = 1.0", "min: float = 0.0"] ->
-        #    def fn(t0, s0: float, s1: bool, max: float = 1.0, min: float = 0.0)
-        # - args_kw is the string of args/kwargs used to call the op, same as args_annot_kw but
-        # without type annotations
+        # - args_kw is the value of args and strings of kwargs used to call the op (without type annotations)
```

Comment:
I think it would be nice to have an example of what is happening (similar to what was present)

Reply:
Done, thanks!